_docs/quick-start.markdown

```diff
@@ -30,19 +30,66 @@ Click **Create** when ready, and the IDE will generate a project with the follow
 
 {% include h title="Open a project" tag="open-prj" %}
 
-Go to `File | Open` and select the folder containing the root `Cargo.toml` file.
+Go to `File | Open` and select the folder containing the root `Cargo.toml` file. Check the Cargo tool window (`View | Tool windows | Cargo`) to make sure your project was imported successfully:
 
-Check the Cargo tool window (`View | Tool windows | Cargo`) to make sure your project was imported successfully:
+![check-cargo-tw](/assets/quick-start/qsg_cargo_tw.png){:width="300px"}
 
-![check-cargo-tw](/assets/quick-start/qsg_cargo_tw.png){:width="350px"}
-
-{% include h title="Clone a repository" tag="open-prj" %}
+{% include h title="Clone a repository" tag="clone-repo" %}
 
 Go to `VCS | Checkout from Version Control` on the main menu or click **Checkout from Version Control** on the welcome screen:
 
-![qsg_clone](/assets/quick-start/qsg_clone.png){:width="350px"}
+![qsg_clone](/assets/quick-start/qsg_clone.png){:width="300px"}
 
 In the case of GitHub, provide the repository URL, test the connection using the **Test** button, and click **Clone**:
 
-![qsg_clone_dialog](/assets/quick-start/qsg_clone_dialog.png){:width="750px"}
+![qsg_clone_dialog](/assets/quick-start/qsg_clone_dialog.png){:width="700px"}
+
+{% include h title="Configure project settings" tag="settings" %}
+
+In `Settings/Preferences | Languages & Frameworks | Rust`, you can adjust the toolchain settings and select the macro expansion engine.
+
+![check-cargo-tw](/assets/quick-start/qsg_mainsettings.png){:width="750px"}
+
+ If you plan to use an external linter, select it in `Settings/Preferences | Languages & Frameworks | Rust | Cargo`:
```

Comment:
Also, I think we should add a note what is external liner and why I may want to use it